### ISSUE TITLE
Can't pass to spacestation selection

### ISSUE THREAD

**Issue by cloud023** [onikamigames/starshipgame#463]
Hi I'd follow all the instructions but can't still passed the spacestation selection this is my bsc address

bsc : 0x9876Fea444860606A56b5a8EfC6eDf6B9DbE264f
wax : ubiyq.wam

**Comment by kamil11-cloud** [comment]
same problem

**Comment by AsterSol** [comment]
same problem

**Comment by tailjef** [comment]
did you already chose the spacestation?

**Comment by ZOnlight** [comment]
same problem

**Comment by cloud023** [comment]
> did you already chose the spacestation?

yes I did multiple times already both in the website UI and wax.blocks website
